src/cmd-prune

```diff
@@ -176,6 +176,7 @@ error_during_pruning = False
 for build in builds_to_delete:
     print(f"Pruning build {build.id}")
     try:
+        subprocess.check_call(['ostree', '--repo=tmp/repo', 'refs', '--delete', build.id])
```

Comment:
WDYT of instead

```suggestion
        subprocess.run(['ostree', '--repo=tmp/repo', 'refs', '--delete', build.id])
```

?

I don't think this should be an error. It's totally possible the whole repo was nuked and that's OK. It's temporary anyway. (Also works around the fact that `rmtree` will never be called if this trips.)

Reply:
`refs --delete` silently does nothing if the ref doesn't exist.

Reply:
Nice.